main.js

```diff
@@ -73,33 +73,64 @@ const displayBook = (id, title, author) => {
       li.remove();
     } else {
       li.remove();
-      bookList.classList.add('border');
     }
   });
 };
 
-if (bookList !== null) {
-  bookList.classList.add('border');
-} else {
-  bookList.classList.remove('border');
+if (books !== null) {
+  books.forEach((book) => {
+    displayBook(book.id, book.title, book.author);
+  });
 }
 
 document.addEventListener('DOMContentLoaded', () => {
   addBtn.addEventListener('click', (e) => {
     e.preventDefault();
     const title = document.getElementById('title').value;
     const author = document.getElementById('author').value;
+    const toTitleCase = (str) => str.toLowerCase().split(' ').map((word) => `${word.charAt(0).toUpperCase()}${word.slice(1)}`).join(' ');
     const id = Date.now();
-    const book = new Book(id, title, author);
+    const book = new Book(id, toTitleCase(title), toTitleCase(author));
     book.addBook();
     if (title && author) {
       displayBook(book.id, book.title, book.author);
     }
   });
 });
 
-if (books !== null) {
-  books.forEach((book) => {
-    displayBook(book.id, book.title, book.author);
-  });
-}
\ No newline at end of file
+const date = document.getElementById('date');
+// eslint-disable-next-line no-undef
```

Comment:
Hello @team, disabling linters is a bad practice in development because linters helps produce organized and readable code. 

- [ ] I suggest that you delete it ☝🏻 out and fix the linter instead.

Reply:
Resolved. Thanks